packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleImageViewer/SingleImageViewerContainer.js

```diff
@@ -37,44 +22,33 @@ class SingleImageViewerContainer extends React.Component {
     })
   }
 
-  async handleSubject () {
-    const { subject } = this.props
+  async getImageSize () {
+    const { onError, subject } = this.props
```

Comment:
`onError` isn't being used in this method anymore. Why did the try/catch get removed? Is it bubbling up and being caught in `onLoad`? The test specs indicate it's being caught, so this isn't a blocker, just more of an understanding question because I find the async/await error handling and bubbling of events to be a harder to follow than promises.

Reply:
Good catch. If we catch the error here, then the calling function will always run successfully. In this case, that means `onReady` always gets called and you get classifications with `{ naturalHeight: 0, naturalWidth: 0 }` if the error is trapped here. I thought it was better to throw an error here and catch it in `onLoad`, so that we can set the store to an error state.